h/admin.py

```diff
@@ -186,6 +191,24 @@ def users_index(request):
 
     return {'username': username, 'user': user, 'user_meta': user_meta}
 
+@view.view_config(route_name='admin_users_delete',
+                 request_method='POST',
```

Comment:
If I were a pernickety asshole, I'd suggest lining up the first character of `request` with the first character of `route` on the line above. As it happens, I'm a pernickety asshole. :)